py/defs.bzl

```diff
@@ -37,6 +38,32 @@ def _py_binary_or_test(name, rule, srcs, main, imports, deps = [], resolutions =
         tags = ["manual"],
     )
 
+def _find_main_impl(ctx):
```

Comment:
Would prefer if this went in its own bzl file to keep this simply wrappers and re-exports.

Reply:
done - used the same filename as the rules_python code to make it a bit easier to reason about